dashboard/test/controllers/scripts_controller_test.rb

```diff
@@ -355,54 +355,33 @@ class ScriptsControllerTest < ActionController::TestCase
     refute Script.find_by_name(script.name).hidden
   end
 
-  test 'updates project_sharing' do
+  test 'update: can update general_params' do
```

Comment:
note because this diff looks confusing: i renamed the test and combined what was previously 2 separate tests ("updates project_sharing" and "updates curriculum_umbrella") so that we can test updating any property in `general_params` (in `scripts_controller.rb`) in the same unit test